aspnetcore/blazor/common/samples/3.x/BlazorSample/README.md

```diff
@@ -4,7 +4,7 @@ This sample illustrates the use of Blazor scenarios described in the Blazor docu
 
 ## Call web API example
 
-The web API example requires a running web API based on the sample app for the <a href="https://docs.microsoft.com/aspnet/core/tutorials/first-web-api">Tutorial: Create a web API with ASP.NET Core MVC</a> topic. The sample app makes requests to the web API at `https://localhost:10000/api/todo`. If a different web API address is used, update the `ServiceEndpoint` constant value in the Razor component's `@functions` block.</p>
+The web API example requires a running web API based on the sample app for the <a href="https://docs.microsoft.com/aspnet/core/tutorials/first-web-api">Tutorial: Create a web API with ASP.NET Core MVC</a> topic. The sample app makes requests to the web API at `https://localhost:10000/api/todo`. If a different web API address is used, update the `ServiceEndpoint` constant value in the Razor component's `@functions` directive.</p>
```

Comment:
OK, I spoke to @rynowak about this and he is fine with the phrase "@code block" or "@functions block". His main concern is to avoid calling the block a "code block", since it contains members not arbitrary code. That's what `@{ }` is for. So we should revert this block->directive changes in this PR.